src/logic/Operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from "big.js";
+
+export default function Operate(numberOne, numberTwo, operation) {
+    const one = Big(numberOne);
+    const two = Big(numberTwo);
+    if (operation == "/"){
+        return one.div(two).toString();
+    }
```

Comment:
- [ ] As other numbers can't be divided by zero there should be a try catch here, to prevent the calculator from crashing.
```js
 if (operation === '/') {
    try {
      return one.div(two).toString();
    } catch (err) {
      return "Can't divide by 0.";
    }
  }
```